packages/@react-stately/table/src/TableCollection.ts

```diff
@@ -234,6 +235,7 @@ export class TableCollection<T> extends GridCollection<T> {
     this.rowHeaderColumnKeys = rowHeaderColumnKeys;
     this.body = body;
     this.headerRows = headerRows;
+    this._size = [...body.childNodes].length;
```

Comment:
```suggestion
    this._size = body.childElementCount;
```
can we use this instead of making a new array to get something I *think* we already have direct access to?

Reply:
This is not a DOM node, so that property doesn't exist. Might be nice to cache it though

Reply:
darn, had hoped that wasn't the case, haha